frontend/src/pages/SessionOverview/SessionOverview.js

```diff
@@ -27,18 +44,41 @@ function SessionOverview() {
           <LinkButton
             name="CREATE NEW SESSION"
             to="/sessionForm"
-            state={{ initialData: INITIAL_SESSION_DATA, action: "NEW_SESSION" }}
+            onClick={() => onCreateNewSession()}
+          />
+
+          <hr className="separatorLine"></hr>
+          <Button
+            name="Show/Close past sessions"
```

Comment:
Maybe labelling the button depending on the current state would provide more information. So e.g. 
`name={showPastSessions ? "Hide past sessions" : "Show past sessions"}`